scipy/sparse/linalg/_eigen/lobpcg/lobpcg.py

```diff
@@ -155,71 +155,79 @@ def lobpcg(
         sparse matrix.  Often called the "stiffness matrix".
     X : ndarray, float32 or float64
-        Initial approximation to the ``k`` eigenvectors (non-sparse). If `A`
-        has ``shape=(n,n)`` then `X` should have shape ``shape=(n,k)``.
-    B : {dense matrix, sparse matrix, LinearOperator, callable object}
-        Optional.
-        The right hand side operator in a generalized eigenproblem.
-        By default, ``B = Identity``.  Often called the "mass matrix".
-    M : {dense matrix, sparse matrix, LinearOperator, callable object}
-        Optional.
-        Preconditioner to `A`; by default ``M = Identity``.
-        `M` should approximate the inverse of `A`.
-    Y : ndarray, float32 or float64, optional.
-        An n-by-sizeY matrix of constraints (non-sparse), sizeY < n.
-        The iterations will be performed in the B-orthogonal complement
-        of the column-space of Y. Y must be full rank.
-    tol : scalar, optional.
-        Solver tolerance (stopping criterion).
+        Initial approximation to the ``k`` eigenvectors (non-sparse).
+        If ``A`` has ``shape=(n,n)`` then ``X`` must have ``shape=(n,k)``.
+    B : {sparse matrix, ndarray, LinearOperator, callable object}
+        Optional. By default ``B = None``, which is equivalent to identity.
+        The right hand side operator in a generalized eigenproblem if present.
+        Often called the "mass matrix". Must be Hermitian positive definite.
+    M : {sparse matrix, ndarray, LinearOperator, callable object}
+        Optional. By default ``M = None``, which is equivalent to identity.
+        Preconditioner aiming to accelerate convergence.
+    Y : ndarray, float32 or float64, optional
+        By default ``Y = None``.
+        An ``n-by-sizeY`` ndarray of constraints with ``sizeY < n``.
+        The iterations will be performed in the ``B``-orthogonal complement
+        of the column-space of ``Y``. ``Y`` must be full rank if present.
+    tol : scalar, optional
         The default is ``tol=n*sqrt(eps)``.
-    maxiter : int, optional.
-        Maximum number of iterations.  The default is ``maxiter=20``.
-    largest : bool, optional.
+        Solver tolerance for the stopping criterion.
+    maxiter : int, optional
+        The default is ``maxiter=20``. Maximum number of iterations.
+    largest : bool, optional. The default is ``largest=True``.
         When True, solve for the largest eigenvalues, otherwise the smallest.
     verbosityLevel : int, optional
-        Controls solver output.  The default is ``verbosityLevel=0``.
-    retLambdaHistory : bool, optional.
-        Whether to return eigenvalue history.  Default is False.
-    retResidualNormsHistory : bool, optional.
-        Whether to return history of residual norms.  Default is False.
+        By default ``verbosityLevel=0`` no output.
+        Controls the solver standard/screen output.
+    retLambdaHistory : bool, optional
+        The default is ``False``.
```

Comment:
Then we should be consistent
```suggestion
    retLambdaHistory : bool, default: False
```